src/replay/trackers.zig

```diff
@@ -101,6 +101,25 @@ pub const SlotTracker = struct {
         }
         return frozen_slots;
     }
+
+    pub fn parents(
+        self: *const SlotTracker,
+        allocator: Allocator,
+        slot: Slot,
+    ) Allocator.Error![]const Slot {
+        var parents_list = std.ArrayListUnmanaged(Slot){};
```

Comment:
Same comment as on the hashmap case, this should use `.empty` initialization.

Reply:
[726e5e5](https://github.com/Syndica/sig/pull/762/commits/726e5e5d8b28353bd631b5fe58ddcd736d59bc1a)